src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -8,6 +8,7 @@
 import java.util.List;
 
 public class ComplexCommand implements ICompoundCommand {
+    private final String name = "ComplexCommand";
```

Comment:
I think that it should be possible to set the name of a command, e.g., through constructor.
The name of the complex command is an element of our design (e.g., it is shown in logger and command manager window). Do you use it in some other way?